practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #45.            edge_predictions = # Apply sigmoid activation to the edge_predictions and scale it
</pre><p>-&gt; scale it to a 1-5 scale</p><p>(best to re-iterate yourself here)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Resolved thanks :) </p>